backend/src/test/groovy/de/qaware/mercury/integrationtest/ImageAnalyzerImplTest.groovy

```diff
@@ -0,0 +1,27 @@
+package de.qaware.mercury.integrationtest
+
+import de.qaware.mercury.business.image.Image
+import de.qaware.mercury.business.image.color.ColorFindResult
+import de.qaware.mercury.business.image.impl.ImageAnalyzerImpl
+import de.qaware.mercury.test.IntegrationTestSpecification
+import org.springframework.beans.factory.annotation.Autowired
+import spock.lang.Subject
+
+class ImageAnalyzerImplTest extends IntegrationTestSpecification {
+
+    @Subject
+    @Autowired
+    ImageAnalyzerImpl sut;
+
+    def "test getDominantColor"() {
+        given:
+        // a 128x128 picture with the uniform color #008081
+        InputStream testImage = getClass().getClassLoader().getResourceAsStream("test/images/2.jpg");
```

Comment:
`ImageAnalyzerImplTest.class.getResourceAsStream("/test/images/2.jpg")` oder `getResourceAsStream("/test/images/2.jpg")`. Dieses getClassLoader() macht in bestimmten Fällen lustige Probleme.

Reply:
Und eigentlich müsstest du den Stream in nem `cleanup:` block noch zumachen.